docs/streams/upgrade-guide.html

```diff
@@ -824,7 +824,7 @@ <h3><a id="streams_api_changes_0102" href="#streams_api_changes_0102">Streams AP
         <li> method <code>extract()</code> has a second parameter now </li>
         <li> new default timestamp extractor class <code>FailOnInvalidTimestamp</code>
             (it gives the same behavior as old (and removed) default extractor <code>ConsumerRecordTimestampExtractor</code>) </li>
-        <li> new alternative timestamp extractor classes <code>LogAndSkipOnInvalidTimestamp</code> and <code>UsePreviousTimeOnInvalidTimestamps</code> </li>
+        <li> new alternative timestamp extractor classes <code>LogAndSkipOnInvalidTimestamp</code> and <code>UsePartitionTimeOnInvalidTimestamps</code> </li>
```

Comment:
This should not be changed, because it's the upgrade guide of the old release. We need to add a new upgrade section for `2.5` release instead.